pkgs/test/default.nix

```diff
@@ -22,6 +22,8 @@ with pkgs;
   cc-wrapper-libcxx-7 = callPackage ./cc-wrapper { stdenv = llvmPackages_7.libcxxStdenv; };
   stdenv-inputs = callPackage ./stdenv-inputs { };
 
+  haskell = callPackage ./haskell { };
```

Comment:
For this to be tested, we also need an entry in pkgs/top-level/release.nix.

Reply:
oh thanks